errors.go

```diff
@@ -42,6 +42,7 @@ var errLog = Logger(log.New(os.Stderr, "[mysql] ", log.Ldate|log.Ltime|log.Lshor
 // Logger is used to log critical error messages.
 type Logger interface {
 	Print(v ...interface{})
+	Printf(format string, v ...interface{})
```

Comment:
Ah, I have surprised, that logger is interface. As I know, in old versions, the standard logger was used.
I think it is not good idea to modify this interface.

Reply:
ok